### ISSUE TITLE
Flaky Test: ghost online ddl

### ISSUE THREAD

**Issue by rohit-nayak-ps** [vitessio/vitess#7834]
#### Overview of the Issue
Seeing this often:
```
=== RUN   TestSchemaChange/successful_online_alter,_vtctl
I0412 20:22:56.818540   16544 vtctlclient_process.go:168] Executing vtctlclient with command: vtctlclient -server localhost:16006 ApplySchema -sql 
		ALTER TABLE vt_onlineddl_test_03
			ENGINE=InnoDB -ddl_strategy gh-ost ks
# Generated UUID (for debug purposes):
<de38838a_9bcc_11eb_9203_000d3a7d4d46>
    onlineddl_ghost_test.go:350: 
        	Error Trace:	onlineddl_ghost_test.go:350
        	            				onlineddl_ghost_test.go:326
        	            				onlineddl_ghost_test.go:214
        	Error:      	"CREATE TABLE `vt_onlineddl_test_03` (
        	            	  `id` bigint(20) NOT NULL,
        	            	  `msg` varchar(64) DEFAULT NULL,
        	            	  `non_online` int(10) unsigned NOT NULL,
        	            	  PRIMARY KEY (`id`)
        	            	) ENGINE=InnoDB DEFAULT CHARSET=utf8" does not contain "ghost_col"
        	Test:       	TestSchemaChange/successful_online_alter,_vtctl

```

**Comment by shlomi-noach** [comment]
My immediate thinking is that this is related to increased runtime on Actions servers, as the problem doesn't reproduce locally. Looking into it.

**Comment by shlomi-noach** [comment]
solved via #7847 . Was an issue of GitHub Actions DNS resolution.
